src/Core/MySQL/MySQLReplication.cpp

```diff
@@ -584,6 +585,19 @@ namespace MySQLReplication
                         }
                         break;
                     }
+                    case MYSQL_TYPE_BIT:
+                    {
+                        UInt32 bits = ((meta >> 8) * 8) + (meta & 0xff);
```

Comment:
Could you please explain what does it mean? What's the difference from `field_len`? Can it be greater that 64?

Reply:
https://github.com/mysql/mysql-server/blob/3290a66c89eb1625a7058e0ef732432b6952b435/sql/log_event.cc#L1968 reference by this. 

The first byte of metadata is an integer divided by 8, and the second byte of metadata is the remainder of 8. size is  equivalently field_len（the value of field_len is not calculated above）

“The BIT data type is used to store bit values. A type of BIT(M) enables storage of M-bit values. M can range from 1 to 64.” reference from: https://dev.mysql.com/doc/refman/8.0/en/bit-type.html